shared/js/pageStore.js

```diff
@@ -0,0 +1,106 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var loop = loop || {};
+loop.store = loop.store || {};
+
+loop.store.PageStore = function() {
+  "use strict";
+
+  /**
+   * Page store.
+   *
+   * @param {loop.Dispatcher} dispatcher  The dispatcher for dispatching actions
+   *                                      and registering to consume actions.
+   * @param {Object}          options     Options object:
+   * - {DataDriver} dataDriver The driver to use for adding or removing the pages.
+   */
+  var PageStore = loop.store.createStore({
+    initialize(options = {}) {
+      if (!options.dataDriver) {
+        throw new Error("Missing option dataDriver");
+      }
+
+      this._dataDriver = options.dataDriver;
+    },
+
+    getInitialStoreState() {
+      return {
+        pages: []
+      };
+    },
+
+    actions: [
+      "addPage",
+      "addedPage",
+      "deletePage",
+      "deletedPage",
+      "updateRoomInfo"
+    ],
+
+    /**
+     * Handle UpdateRoomInfo action by saving the id for the current user.
+     */
+    updateRoomInfo({ userId }) {
+      this._currentUserId = userId;
```

Comment:
Not sure if we want both `userId` and `participantName` but we'll definitely want the name to show the letter on the tile.